images-list

```diff
@@ -313,6 +318,8 @@ library/nginx rancher/mirrored-library-nginx 1.19.2-alpine
 library/nginx rancher/mirrored-library-nginx 1.19.9-alpine
 library/nginx rancher/mirrored-library-nginx 1.21.0-alpine
 library/nginx rancher/mirrored-library-nginx 1.21.1-alpine
+library/nginx rancher/mirrored-library-nginx 1.21.6-alpine
+library/registry rancher/mirrored-registry 2.7.1
```

Comment:
@andreas-kupries this should be `rancher/mirrored-library-registry` for the same reason as above for `library/nginx`; we need another EIO issue created for this

Reply:
Yep. Done. Commit 7722deff7ffc1605eae25b4ecbd45a7cf42480a4